src/applications/gi/components/heading/GiBillBreadcrumbs.jsx

```diff
@@ -36,9 +36,13 @@ class GiBillBreadcrumbs extends React.Component {
     if (pathname.match(/profile/)) {
       if (this.props.includeSearch) {
         crumbs.push(
-          <a onClick={browserHistory.goBack} key="search-results">
+          <button
```

Comment:
A button here solves the keyboard focus problem, though it could be confusing to screen reader users. I like this as an immediate fix, because it allows for keyboard focus. Let's keep [#15378](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15378) open to refactor to a link that remembers the search query, like we did for Facility Locator.